src/main/java/sirius/db/mixing/FieldLookupCache.java

```diff
@@ -28,8 +28,7 @@
  */
 @Register(classes = FieldLookupCache.class)
 public class FieldLookupCache {
-
-    private Cache<String, Value> cache = CacheManager.createLocalCache("mixing-field-lookup");
+    protected Cache<String, Value> cache = CacheManager.createLocalCache("mixing-field-lookup");
```

Comment:
Why can't this stay private?

Reply:
Ah yes, with the new reflection call this should be possible